thread 'rustc' panicked at 'called `Result::unwrap()` on an `Err` value: (MoveData { move_paths: [MovePath { place: _0 }
error: internal compiler error: unexpected panic
note: the compiler unexpectedly panicked. this is a bug.